nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/pandar_128e4x.hpp

```diff
@@ -80,7 +83,7 @@ class Pandar128E4X : public HesaiSensor<hesai_packet::Packet128E4X>
 
     int channel_offset_ns;
     bool is_hires_mode = packet.tail.operational_state == OperationalState::HIGH_RESOLUTION;
-    auto azimuth_state = packet.tail.geAzimuthState(block_id);
+        auto azimuth_state = packet.tail.geAzimuthState(block_id);
```

Comment:
Unneccesary tab?

Reply:
Addressed in [adf43fe](https://github.com/tier4/nebula/pull/126/commits/adf43fef5af5924675b60e317a6de3b59e7a9f06)